ObjectCreation/BoxCreator.FCMacro

```diff
@@ -17,8 +17,7 @@ __Files__ = 'boxcreator/__init__.py,boxcreator/boxcreator.py'
 
 # Form implementation generated from reading ui file 'boxcreator.ui'
 #
-# Created: Thu Oct 25 21:10:42 2018
-#      by: pyside-uic 0.2.15 running on PySide 1.2.2
+# Created by: PyQt4 UI code generator 4.12.1
```

Comment:
Would it be possible to dynamically load the `ui` file rather than generating and including the Python equivalent to it? Cf. the [wiki](https://github.com/FreeCAD/FreeCAD-macros/wiki/Dynamically-load-graphical-interface-definition).

Reply:
I need some help to do this.
d = gui.PySideUic.loadUiType('boxcreator/boxcreator.ui')
gives me an error message <class 'FileNotFoundError'>: [Errno 2] File or directory not found: 'boxcreator/boxcreator.ui'
Is there a FCMacro that uses the .ui import that I can watch as an example ?

Reply:
Yes, cf. link above.

Reply:
I tried to use the .ui file in the same way you do it in the FEM example, but the UI does not show up. If I replace my boxcreator.ui file fem_animate_mode_shapes.ui, a UI does show up. What is wrong with my .ui file ?
Can you please have a look at [my last commit](https://github.com/chbergmann/FreeCAD-macros/blob/boxcreator/ObjectCreation/BoxCreator.FCMacro) ?

Reply:
The trick was to assign `Boxcreator_Ui_Dialog()` to a variable `d = Boxcreator_Ui_Dialog()`. What I don't understand is why this is not needed for `FemAnimateModeShapes`.

I integrated you work

Reply:
thanks